docs/cloud/rahti2/usage/projects_and_quota.md

```diff
@@ -181,51 +186,9 @@ Note: `m` stands for milicores. `500m` will be the equivalent of 0.5 cores, or i
 
 Rahti 2 enforces a maximum limit/request ratio of 5. This means that the CPU or memory `limits` cannot be more than 5 times the `request`. So if the CPU request is 50m, the CPU limit cannot be higher than 500m. And if we wanted to increase the CPU limit to 1, we will have to increase as well the request to at least 100m.
 
-## Cluster Quotas
-
-In addition to the project resource quota, cluster resource quota is in use. This quota is shared by multiple projects created by the same user. This means that cluster quota is enforced per user. A single user will be able to request as many projects as it is necessary, but the total resource use of all these projects cannot exceed the cluster quota.
-
-Initial cluster quotas are 5 times of the project resource quota.
-
-| Resource type | Quota |
-| --- | --- |
-| limits.cpu | 20 |
-| limits.memory | 80Gi |
-| request.storage | 500Gi |
-| limits.ephemeral-storage | 25Gi |
-| openshift.io/imagestreams | 100 |
-| persistentvolumeclaims | 25 |
-| pods | 500 |
-
-You can check your current cluster quota by running this command line:
-
-```sh
-$ oc describe AppliedClusterResourceQuota
-
-Name:		crq-XXXXXXXX
-Created:	19 hours ago
-Labels:		<none>
-Annotations:	<none>
-Namespace Selector: ["test"]
-Label Selector:
-AnnotationSelector: map[openshift.io/requester:XXXXXXXX]
-Resource			Used	Hard
---------			----	----
-limits.cpu			0	20
-limits.ephemeral-storage	0	25Gi
-limits.memory			0	80Gi
-openshift.io/imagestreams	1	100
-persistentvolumeclaims		0	25
-pods				0	500
-requests.storage		0	500Gi
-```
-
-It not only shows the Quota, but also the current Usage.
-
 ## Requesting more quota
 
-If you need more resources that the defaults, you can apply for more quota by contacting the Service Desk. See the [Contact page](../../../support/contact.md) for instructions. Quota requests are
-handled on a case-by-case basis depending on the currently available resourcesin Rahti 2 and the use case.
+If you need more resources that the defaults, you can apply for more quota by contacting the Service Desk. See the [Contact page](../../../support/contact.md) for instructions. Quota requests are handled on a case-by-case basis depending on the currently available resources in Rahti 2 and the use case.
 
 ## Sharing projects with other users
```

Comment:
Is this section useful? When Rahti is enabled on a CSC computing project, it automatically applies `cscrolebinding-{rahti-project}-{CSC_project_number}` as a RoleBinding

Reply:
I agree, keep the my CSC instructions